www/src/components/SectionTitle.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
+
+type SectionTitleProps = {
+  readonly title: string;
+  readonly description: string;
+};
+
+export const SectionTitle: FC<SectionTitleProps> = ({ title, description }) => {
```

Comment:
There are other sections in the design that will also have a title and a description with the same styling, hence why I opted into making a component for that. It will be used in the future with other sections as well.